main.go

```diff
@@ -119,6 +119,8 @@ var (
 	nodeID          = flag.String("nodeid", "", "node id")
 	runAsController = flag.Bool("run-as-controller", false, "Only run as controller service")
 	driver          = flag.String("driver", TypePluginDISK, "CSI Driver")
+	clusterID       = flag.String("clusterid", "", "cluster id")
+	aliUID          = flag.String("aliuid", "", "aliyun user id")
```

Comment:
Should we use env? Env seems to be easier for kustomize.
aliUID is also available at `http://100.100.100.200/latest/meta-data/owner-account-id` or `http://100.100.100.200/latest/dynamic/instance-identity/document`